OpenCyphalPicoBase-firmware.ino

```diff
@@ -804,16 +815,14 @@ ExecuteCommand::Response_1_1 onExecuteCommand_1_1_Request_Received(ExecuteComman
   }
   else if (req.command == ExecuteCommand::Request_1_1::COMMAND_FACTORY_RESET)
   {
-    /* set factory settings */
-    update_period_ms_inputvoltage=3*1000;
-    update_period_ms_internaltemperature=10*1000;
-    update_period_ms_input0=500;
-    update_period_ms_input1=500;
-    update_period_ms_input2=500;
-    update_period_ms_input3=500;
-    update_period_ms_analoginput0=500;
-    update_period_ms_analoginput1=500;
-    update_period_ms_light=250;
+    /* erasing eeprom by writing FF in every cell */
+    size_t const NUM_PAGES = eeprom.device_size() / eeprom.page_size();
+    for(size_t page = 0; page < NUM_PAGES; page++)
+    {
+      uint16_t const page_addr = page * eeprom.page_size();
+
+      eeprom.fill_page(page_addr, 0xFF);
```

Comment:
If I were you, I'd be calling the watchdog in here to prevent a pre-mature reset. Not calling the watchdog in here is probably what triggers the reset you have observed.

```diff
+rp2040.wdt_reset();
```